specs/DesignDiscussions/Carousel.md

```diff
@@ -108,6 +108,49 @@ In order to meet our schedule, we need to avoid some common pain points that wou
 
 All elements not mentioned above are allowed inside a carousel.
 
+## Header
+![image](https://user-images.githubusercontent.com/4112696/183519757-156a18a9-73e7-47d9-8e00-ad84d0070f99.png)
+
+Header is an optional property whose value is an array of TextBlocks. Header is positioned above the carousel page, and the position of the text is static.
+When carousel page transitions to a new page, the text shall remain. Each subsequent `TextBlock` is laid below a previous `TextBlock` as `TextBlock`s in `body` of `AdaptiveCards` would be rendered.
+
+
+Header can be used for purely decorative purpose as well as a11y header.
+Author of card can designate a TextBlock in the array to be a11y header by setting its `style` property to `heading`. 
+Renderers shall apply a11y heading role according to their respective a11y options. 
+
+Header supports an implied type. The implied type of `header` is `TextBlock`.
+As a result, author can shortcircuits specifying the type. When this happens, renderers shall assume the type `TextBlock` and renders the content accordingly.
+Implied type shall be remained `TextBlock` even when additional types are added to the list of supported types in the future.
+```
+{
+  "type": "Carousel",
+  "header": [
```

Comment:
If we need to support rtl,padding,spacing,isVisible and all other properties a container have for the header, you may need to change the header to a container object instead of a list of text controls

Reply:
Hi @licanhua ,
`header` behaves similar to `body` and `action` of `AdaptiveCard`. Elements in `header` inherits rtl, iVisible, spacing and padding of Carousel as `action` and `body` does with `AdaptiveCard`. Author can also configure them for each element in `header`.
In the future, we can add Container as an allowable elements if there is need for it, then authors can add TextBlocks to the Container.

Reply:
@JeanRoca, @paulcam206 and @jwoo-msft 
Let's make padding as an example.
Assume orange is the head area, and not colored area is the Carousel.

The padding setting of Carousel applies to the green area.
But we don't have setting for the yellow part because we can't set padding to body and header separately.

![image](https://user-images.githubusercontent.com/6290692/184388954-87b76fb9-8189-437e-b8f2-08c53eb8cc48.png)

Reply:
@licanhua,

AdaptiveCards do not let card author to set padding explicitly. padding is applied implicitly by style rule. Another example of `padding` is an enumeration value of spacing which is applied between adaptive elements.

Reply:
I leave this to @paulcam206 and @JeanRoca since they have better understanding on customer requirements and the product. 
Padding here is just an example to illustrate the problem. The question is: will we need to set proprieties to header area and carousel area separately with different value? 

For example (It's a fiction case and don't make too much sense in reality), carousel is set to rtl=false, while customer want header area be rtl=true. Because we can't set rtl on header itself, we have to set rtl=true on all elements in the header.

Reply:
resolved in the design discussion meeting.